styx-scheduler-service/src/test/java/com/spotify/styx/state/handlers/FlyteRunnerHandlerTest.java

```diff
@@ -164,7 +164,8 @@ public void shouldPollInRunning() throws FlyteRunner.PollingException {
 
     flyteRunnerHandler.transitionInto(runState, eventRouter);
 
-    verify(flyteRunner).poll(getFlyteExecutionId(FLYTE_EXECUTION_DESCRIPTION, EXECUTION_ID), runState);
+    verify(flyteRunner).poll(getFlyteExecutionId(FLYTE_EXECUTION_DESCRIPTION,
+        new StringBuilder(EXECUTION_ID).reverse().toString()), runState);
```

Comment:
You can use the reverse function that it is already defined in line 63
```suggestion
        reverse.apply(EXECUTION_ID)), runState);
```

Reply:
yes i did forgot to push this change. It's already applied on my local branch